test/sagemaker_tests/pytorch/inference/integration/local/test_serving.py

```diff
@@ -62,6 +83,7 @@ def fixture_test_loader():
 def test_serve_json_npy(
     test_loader, use_gpu, docker_image, framework_version, sagemaker_local_session, instance_type
 ):
+    env = disable_token_auth(framework_version)
```

Comment:
For these, can change this to extend multiple environment var setting?
Something like
```
env = {}
env.update(set_disable_token_auth(framework_verison))
```

That way, if in the future we need to update multiple environment, we can extend the dictionary easily.